code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -304,7 +323,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 /datum/game_mode/dynamic/set_round_result()
 	// If it got to this part, just pick one high impact ruleset if it exists
 	for(var/datum/dynamic_ruleset/rule in executed_rules)
-		if(rule.flags & HIGH_IMPACT_RULESET)
+		if(CHECK_BITFIELD(rule.flags, HIGH_IMPACT_RULESET))
```

Comment:
Ah. actually, this could be a potential bug. Maybe change this to NO_OTHER_ROUNDSTARTS, since that is more consistent with the type of ruleset to have its own round result. Unrelated to this PR though.

Some things have high impact but no round_result. Depends.

Reply:
Fixed